staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/pruning/algorithm.go

```diff
@@ -18,37 +18,16 @@ package pruning
 
```

Comment:
I wasn't expecting the changes to this package... are these intended to be no-op changes to refactor/share the unknown path accumulation logic, or is this changing behavior of this package?

I was hoping the change would be contained to the unstructured metadata function